test/test_doctor_command.rb

```diff
@@ -33,4 +33,22 @@ class TestDoctorCommand < JekyllUnitTest
       assert_includes output, "Warning: The following URLs only differ by case. On a case-insensitive file system one of the URLs will be overwritten by the other: #{dest_dir}/about/index.html, #{dest_dir}/About/index.html"
     end
   end
+
+  context 'Liquid errors' do
+    setup do
+      clear_dest
+    end
+
+    should 'return error for invalid Liquid' do
+      @site = Site.new(Jekyll.configuration({
+        "source" => File.join(source_dir, '/_liquid_errors'),
+        "destination" => dest_dir
+      }))
+      @site.process
+      output = capture_stderr do
+         ret = Jekyll::Commands::Doctor.liquid_errors(@site)
+         assert_equal true, ret
+      end
```

Comment:
That's not good! If there's no output, then the user has no idea why the build failed.